ScoutSuite/providers/azure/resources/sql/databases.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+from azure.mgmt.sql import SqlManagementClient
+
+from ScoutSuite.providers.base.configs.resource_config import ResourceConfig
+
+from .database_blob_auditing_policies import DatabaseBlobAuditingPoliciesConfig
+from .database_threat_detection_policies import DatabaseThreatDetectionPoliciesConfig
+from .replication_links import ReplicationLinksConfig
+from .transparent_data_encryptions import TransparentDataEncryptionsConfig
+
+
+class DatabasesConfig(ResourceConfig):
+    # the following static methods will be used to parse the config of each resource nested in a database config
+    # (defined in 'children' attribute below):
+    @staticmethod
+    def _parse_auditing_config(config):
+        return {
+            'auditing_enabled': config.value.state == "Enabled"
+        }
+
+    @staticmethod
+    def _parse_threat_detection_config(config):
+        return {
+            'threat_detection_enabled': config.value.state == "Enabled"
+        }
+
+    @staticmethod
+    def _parse_replication_links_config(config):
+        return {
+            'replication_configured': len(config.value) > 0
+        }
+
+    @staticmethod
+    def _parse_transparent_data_encryption_config(config):
+        return {
+            'transparent_data_encryption_enabled': config.value.status == "Enabled"
+        }
+
+    # register children resources:
+    children = {
+        'Database Auditing Settings': (DatabaseBlobAuditingPoliciesConfig, _parse_auditing_config),
+        'Database Threat Detection Policies': (DatabaseThreatDetectionPoliciesConfig, _parse_threat_detection_config),
+        'Replication Links': (ReplicationLinksConfig, _parse_replication_links_config),
+        'Transparent Data Encryptions': (TransparentDataEncryptionsConfig, _parse_transparent_data_encryption_config)
+    }
```

Comment:
They're not necessary indeed, it was just for a purpose of documentation. Although, they could be useful for printing/debugging purposes, don't you think?

Reply:
I see. I'm usually not a fan of leaving unused stuff in case we want to use it. I guess we're refactoring and this is going to be evolving, so I'll let this one slip. 🤸‍♂️

Reply:
I'm not a fan of leaving unused stuff either. Tbh the primary purpose of those keys was to document my code but since you found that redundant, let's just remove them!